packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -595,7 +594,8 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
   }
 
   /** Emit an event that triggers a UI re-render */
-  private emitTorrentUpdated(infoHash) {
+  private emitTorrentUpdated(infoHash, trigger) {
```

Comment:
Should `trigger` have a type annotation?

Reply:
Good call, resolved in 3e9eb20d1.